compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -796,7 +796,9 @@ public enum DiagnosticErrorCode implements DiagnosticCode {
     MODULE_GENERATED_FOR_CLIENT_DECL_CANNOT_HAVE_MUTABLE_STATE(
             "BCE4044", "module.generated.for.client.decl.cannot.have.mutable.state"),
     CANNOT_IMPORT_MODULE_GENERATED_FOR_CLIENT_DECL(
-            "BCE4045", "cannot.import.module.generated.for.a.client.decl")
+            "BCE4045", "cannot.import.module.generated.for.a.client.decl"),
+    CANNOT_INFER_TYPEDESC_ARGUMENT_OR_EXPECTED_CONTEXTUALLY_EXPECTED_TYPE("BCE4046",
```

Comment:
How is OR correct here? Shall we say `CANNOT_INFER_TYPEDESC_ARGUMENT_WITHOUT_CET`?

Reply:
Updated add0886cbda0a9bdb4d6e1c43292943d414352f4